### ISSUE TITLE
Cannot run dronesploit 

### ISSUE THREAD

**Issue by applecounterfeit** [dhondta/python-tinyscript#13]
```
Traceback (most recent call last):
  File "./main.py", line 2, in <module>
    from tinyscript import *
  File "/usr/local/lib/python3.7/dist-packages/tinyscript/__init__.py", line 11, in <module>
    from .loglib import *
  File "/usr/local/lib/python3.7/dist-packages/tinyscript/loglib.py", line 23, in <module>
    logging.addLogLevel("interact", "cyan", 100)
  File "/usr/local/lib/python3.7/dist-packages/tinyscript/preimports/log.py", line 54, in addLogLevel
    attrs['bold'] = coloredlogs.CAN_USE_BOLD_FONT
AttributeError: module 'coloredlogs' has no attribute 'CAN_USE_BOLD_FONT'
```

**Comment by dhondta** [comment]
OK, this is a [recent issue](https://github.com/xolox/python-coloredlogs/issues/82) originating from [this commit](https://github.com/xolox/python-coloredlogs/commit/044402534c2dd02ff340df2950e485ce4999386c).
I will solve it now in Tinyscript.

**Comment by dhondta** [comment]
This should now be solved.
Please `pip install tinyscript --upgrade`.

**Comment by boredafthohaha** [comment]
I'm getting the same issue, even after upgrading

**Comment by dhondta** [comment]
@boredafthohaha This shall have been fixed with a0e964fba40eeff385df7ea7aad3aae4de28e144. Are you sure you get the exact same error ?
